### ISSUE TITLE
Loss does not decrease beyond 4.332

### ISSUE THREAD

**Issue by egejac** [OlafenwaMoses/ImageAI#603]
I am training my own custom dataset. Upon training I tried batch sizes of  4, 8 and 16. The losses after 100-200 experiments for each batch size are as follows.

batch size = 4           : loss = 4.332
batch size = 8           : loss = 8.431
batch size = 16         : loss = 9. 486

The accuracy with the test data were 74%, 72% and 70% respectively.

How can I decrease the loss?
How do I in that way increase the accuracy?

Thanks

**Comment by OlafenwaMoses** [comment]
- What type of model did you train?
- How many images did you provide for training and test per object?

**Comment by ejgeo** [comment]
I used a YoloV3 model to train.
My objective was to train for only one object. I provided 2632 images to train and 300 images for testing.
